packages/sit-onyx/src/composables/useCustomValidity.ts

```diff
@@ -5,11 +5,13 @@ import enUS from "../i18n/locales/en-US.json";
 import { injectI18n } from "../i18n";
 import type { InputType } from "../components/OnyxInput/types";
 
+export type CustomErrorType = string | FormErrorMessages;
```

Comment:
Should we just change it to always be an object with required shortMessage and optional longMessage? Then we don't have to always check for both cases

Reply:
it also is used for radio button, there it only ever appears in a tooltip so it would be wasteful effort to expect from the devs to provide an object